src/Administration/Resources/app/administration/src/module/sw-cms/blocks/image/image-two-column/index.ts

```diff
@@ -24,8 +24,8 @@ Shopware.Service('cmsService').registerCmsBlock({
     defaultConfig: {
         marginBottom: '20px',
         marginTop: '20px',
```

Comment:
Should we migrate the top and bottom margins to rem values that also match one of the Bootstrap gutters/spacers? e.g. 1.5rem. Then we would not have this "magic" spacing values. https://getbootstrap.com/docs/5.3/layout/gutters/#horizontal-gutters

Sadly, there seems to be no native CSS variable for those spacers.

Reply:
I would not add this now, because it would blow up the scope. But could be something we look into, when improving the CMS with the new component system and maybe a new grid type block, that makes full use of the grid system.